crates/kornia-dnn/src/rtdetr.rs

```diff
@@ -0,0 +1,174 @@
+//! # RT-DETR
+//!
+//! This module contains the RT-DETR model.
+//!
+//! The RT-DETR model is a state-of-the-art object detection model.
+
+use std::path::PathBuf;
+
+use crate::error::DnnError;
+use crate::Detection;
+use kornia_core::{CpuAllocator, Tensor};
+use kornia_image::Image;
+use ort::{GraphOptimizationLevel, Session};
+
+/// Builder for the RT-DETR detector.
+///
+/// This struct provides a convenient way to configure and create an `RTDETRDetector` instance.
+pub struct RTDETRDetectorBuilder {
+    /// Path to the RT-DETR model file.
+    pub model_path: PathBuf,
+    /// Path to the ONNX Runtime dynamic library.
+    pub ort_dylib_path: PathBuf,
```

Comment:
Should be removed along with default `load-dynamic`, instead add docs to instruct the user to use [`ort::init_from`](https://docs.rs/ort/2.0.0-rc.4/ort/fn.init_from.html)

Reply:
you mean force the user to `export ORT_DYLIB_PATH=/path/to/onnxruntime/build/Linux/RelWithDebInfo/libonnxruntime.so` -- as I'm seeing that ` ort::init_from` takes the dyn lib path

Reply:
Either works - `ort::init_from` functions the same as setting `ORT_DYLIB_PATH` and then running `ort::init`, but I have ran into a few instances where it, for whatever reason (likely my fault 🙃), ignored the value set from `std::env::set_var`. If the user wants to set the path programmatically I'd advise them to use `ort::init_from` rather than `std::env::set_var`.

Reply:
done